lab_build/page-components/navbar.html

```diff
@@ -1,45 +1,16 @@
-<!-- Navigation -->
-<nav class="navbar navbar-default " style="background-color: white !important; margin-bottom: 0px !important;">
-    <div class="container-fluid">
-	            <!-- Brand and toggle get grouped for better mobile display -->
-	            <div class="navbar-header page-scroll">
-	                <button type="button" class="navbar-toggle" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1">
-	                    <span class="sr-only">Toggle navigation</span>
-	                    <span class="icon-bar" style="background-color: #77BB41"></span>
-	                    <span class="icon-bar" style="background-color: #77BB41"></span>
-	                    <span class="icon-bar" style="background-color: #77BB41"></span>
-	                </button>
-	                <a class="navbar-brand main-logo-a" href="http://vlab.co.in"><img src="images/logo-new.png" class="img-responsive main-logo" /></a>
-	            </div>			
-	                        
-	            <!-- Collect the nav links, forms, and other content for toggling -->
-	            <div class="collapse navbar-collapse menu-div" id="bs-example-navbar-collapse-1" style="border: 0px;">
-	               
-	                <ul class="nav navbar-nav navbar-right menu-ul search-ul" style="position: relative; top: -5px;">
-	                	<li>
-	                		<form method="get" action="search-result.html">
-	                        	<!-- <input type="text" class="search-textbox" placeholder="SEARCH FOR A LAB" name="q" id="q"/>
-	                        	<input type="submit" value="" class="search-button" /> -->
-	                        </form>
-	                	</li>
-	                </ul>
-	                
-	                <ul class="nav navbar-nav navbar-right menu-ul">
-	                    <li class="page-scroll menu-li " >
-	                        <a href="http://vlab.co.in" class="menu-a" >HOME</a>
-	                    </li>
-
-	                    <li class="page-scroll menu-li " >
-	                        <a href="http://vlab.co.in/participating-institutes" class="menu-a" >PARTNERS</a>
-	                    </li><li class="page-scroll menu-li " >
-	                        <a href="http://vlab.co.in/contact-us" class="menu-a" >CONTACT</a>
-	                    </li>
-	                </ul>
-	                
-	               
-	                	
-	            </div>
-	            <!-- /.navbar-collapse -->  
-    </div>
-    <!-- /.container-fluid -->
-</nav>
\ No newline at end of file
+<ul class="navbar-nav ml-auto text-center d-flex flex-md-row">
```

Comment:
has the navbar been replaced by the ratings components?

Reply:
The earlier navbar.html was static i.e. it didn't used to get built from a handlebar. Since we wanted to have the rating display component on it we create a handlebar for navbar where we have passed the rating component as well as the links for HOME, PARTNERS and CONTACT as list items similar to the way in the old navbar. The resultant navbar looks the same as before we integrated the rating component.